package.json

```diff
@@ -3,23 +3,16 @@
   "version": "0.0.6",
   "description": "Custom element (web component) for the YouTube player.",
   "type": "module",
-  "main": "dist/youtube-video-element.js",
+  "main": "index.js",
   "files": [
-    "./dist/*",
+    "index.js",
     "README.md"
   ],
   "scripts": {
-    "build": "rm -rf dist && rollup --config",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "publish-patch": "yarn run build && np patch --no-tests"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/muxinc/youtube-video-element.git"
-  },
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org"
+    "dev": "npx serve .",
```

Comment:
I think it would be better to make `serve` a dev dep rather than invoking it via npx each time.